scripts/run-gradually-deprecated.sh

```diff
@@ -115,7 +121,7 @@ function checkForUnclearErrorMessages {
 
 function runDeprecatedFunctions {
   echo "==> Checking for use of deprecated functions..."
-  result=$(grep -Ril "d.setid(\"\")" ./internal/services/**/data_source_*.go)
+  result=$(grep -Ril "d.setid(\"\")" ./internal/services/**/*.go | grep data_source)
```

Comment:
This would perhaps be better as a shell glob to avoid bad matches?
```suggestion
  result=$(grep -Ril "d.setid(\"\")" ./internal/services/**/*data_source*.go)
```

Reply:
(same here)